newIDE/app/src/InstancesEditor/SelectionRectangle.js

```diff
@@ -71,6 +109,41 @@ export default class SelectionRectangle {
         this._instancesInSelectionRectangle.push(instance);
       }
     };
+    this.temporarySelector = new gd.InitialInstanceJSFunctor();
+    // $FlowFixMe - invoke is not writable
+    this.temporarySelector.invoke = instancePtr => {
+      // $FlowFixMe - wrapPointer is not exposed
+      const instance = gd.wrapPointer(instancePtr, gd.InitialInstance);
```

Comment:
Is it O(number of instance in the scene)?
How smooth it behave with 1,000 instances?

Reply:
I think this is a generic issue with how instances are stored - could be solved but not trivially/not in this PR.